pandas/tests/test_downstream.py

```diff
@@ -133,10 +133,14 @@ def test_seaborn():
 
 def test_pandas_gbq(df):
 
-    pandas_gbq = import_module("pandas_gbq")  # noqa
+    pandas_gbq = import_module("pandas_gbq")  # noqa:F841
 
 
-@pytest.mark.xfail(reason="0.8.1 tries to import urlencode from pd.io.common")
+@pytest.mark.xfail(
+    raises=ValueError,
+    reason="The Quandl API key must be provided either through the api_key "
+    "variable or through the environmental variable QUANDL_API_KEY",
+)
```

Comment:
@bashtage suggestions for how to re-write this test?

Reply:
Don't test QUANDL endpoint, whcih needs and API key.  Better to test FRED which is open access.